artscraper/googleart.py

```diff
@@ -164,11 +172,13 @@ def save_image(self, img_fp=None, link=None):
 
         img_fp = self._convert_img_fp(img_fp, suffix=".png")
 
-        if self.skip_existing and img_fp.is_file():
+        if self.skip_existing and img_fp.is_file() and img_fp.stat().st_size!=0:
```

Comment:
Is it really a size of 0? If I remember correctly, it was an actual image, just completely white, but maybe that has changed.